plugins/obs-outputs/rtmp-av1.c

```diff
@@ -564,15 +600,15 @@ static void serialize_av1_data(struct serializer *s, const uint8_t *data,
 		case AV1_OBU_TEMPORAL_DELIMITER:
 		case AV1_OBU_REDUNDANT_FRAME_HEADER:
 		case AV1_OBU_TILE_LIST:
-			if (state == START_FOUND)
-				state = END_FOUND;
 			break;
+		case AV1_OBU_FRAME:
+		case AV1_OBU_FRAME_HEADER:
+			compute_av1_keyframe_priority(buf + start_pos,
+						      is_keyframe, priority);
+#ifdef __GNUC__
+			__attribute__((fallthrough));
+#endif
```

Comment:
It seems we normally use this instead:
```c
/* Falls through. */
```
or
```c
/* FALLTHROUGH */
```

Reply:
If we ever get C23 we can do `[[fallthrough]]` but until then it's either a comment or this. I don't think it matters.

Reply:
It doesn't matter too much, I was just looking for consistency because I didn't want to have to try to find all the different variants we used along the way if/when we switch to C23.